build-ps/build-binary.sh

```diff
@@ -479,63 +468,6 @@ fi
        echo "Packaging the test files"
        cp -R $SOURCEDIR/percona-xtradb-cluster-tests $TARGETDIR/usr/local/$PRODUCT_FULL_NAME/
     ) || exit 1
-||||||| merged common ancestors
-    rm -rf "$WORKDIR_ABS/bld"
-    mkdir "$WORKDIR_ABS/bld"
-    cd "$WORKDIR_ABS/bld"
-
-    cmake $SOURCEDIR ${CMAKE_OPTS:-} -DBUILD_CONFIG=mysql_release \
-        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-RelWithDebInfo} \
-        $DEBUG_EXTRA \
-        -DFEATURE_SET=community \
-        -DCMAKE_INSTALL_PREFIX="/usr/local/$PRODUCT_FULL" \
-        -DMYSQL_DATADIR="/usr/local/$PRODUCT_FULL/data" \
-        -DCOMPILATION_COMMENT="$COMMENT" \
-        -DWITH_PAM=ON \
-        -DWITH_ROCKSDB=ON \
-        -DWITH_INNODB_MEMCACHED=ON \
-        -DWITH_ZLIB=system \
-        -DWITH_NUMA=ON \
-        -DWITH_LDAP=ON \
-        -DDOWNLOAD_BOOST=1 \
-        -DFORCE_INSOURCE_BUILD=1 \
-        -DWITH_LIBEVENT=bundled \
-        -DWITH_ZSTD=bundled \
-        -DWITH_BOOST="$WORKDIR_ABS/libboost" \
-        $WITH_MECAB_OPTION $OPENSSL_INCLUDE $OPENSSL_LIBRARY $CRYPTO_LIBRARY
-
-    make $MAKE_JFLAG $QUIET
-    make DESTDIR="$INSTALLDIR" install
-=======
-    rm -rf "$WORKDIR_ABS/bld"
-    mkdir "$WORKDIR_ABS/bld"
-    cd "$WORKDIR_ABS/bld"
-
-    cmake $SOURCEDIR ${CMAKE_OPTS:-} -DBUILD_CONFIG=mysql_release \
-        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-RelWithDebInfo} \
-        $DEBUG_EXTRA \
-        -DFEATURE_SET=community \
-        -DCMAKE_INSTALL_PREFIX="/usr/local/$PRODUCT_FULL" \
-        -DMYSQL_DATADIR="/usr/local/$PRODUCT_FULL/data" \
-        -DROUTER_INSTALL_LIBDIR="/usr/local/$PRODUCT_FULL/lib/mysqlrouter/private" \
-        -DROUTER_INSTALL_PLUGINDIR="/usr/local/$PRODUCT_FULL/lib/mysqlrouter/plugin" \
-        -DCOMPILATION_COMMENT="$COMMENT" \
-        -DWITH_PAM=ON \
-        -DWITH_ROCKSDB=ON \
-        -DWITH_INNODB_MEMCACHED=ON \
-        -DWITH_ZLIB=system \
-        -DWITH_NUMA=ON \
-        -DWITH_LDAP=system \
```

Comment:
This diff changed the `WITH_LDAP` argument, which is missing from PXC. We should add it, as we already received requests for using LDAP auth with pxc, so at least we should test that it works in development builds.

Reply:
Just by building it, it won't become part of the release package automatically, so enabling it shouldn't cause any issues. And then we can create a jira ticket for testing it, but as this is just an authentication plugin, it shouldn't cause any issues.

Reply:
Adding it as "-DWITH_LDAP=system", as far as I can tell from cmake, it's either system or a path to a custom ldap build.